www/includes/easyparliament/templates/html/alert/postcode.php

```diff
@@ -0,0 +1,84 @@
+<div class="full-page alerts-header alerts-header--jumbo">
+    <div class="full-page__row">
+        <div class="full-page__unit">
+
+            <h1>Track your <?php if ($data['recent_election']): ?>new<?php endif ?> MP&rsquo;s parliamentary activity</h1>
+
+          <?php if (isset($data['confirmation_sent'])): ?>
+
+            <div class="alerts-message alerts-message--confirmation-sent">
+                <h2>Almost there!</h2>
+                <p>We just need to check your email address.</p>
+                <p>Please click the link in the email we just sent you.</p>
+            </div>
+
+          <?php elseif (isset($data['signedup_no_confirm']) || isset($data['confirmation_received'])): ?>
+
+            <div class="alerts-message alerts-message--confirmation-received">
+                <h2>Thanks for subscribing!</h2>
+                <p>You will now receive alerts when your MP speaks in Parliament or receives an answer to a written question.</p>
+                <?php if (isset($data['user_signed_in'])): ?>
+                <p><a href="/alert/" class="button radius">Show my email settings</a></p>
+                <?php endif ?>
+            </div>
+
+          <?php elseif (isset($data['error'])): ?>
+            <div class="alerts-message alerts-message--error">
+                <h2>Something went wrong</h2>
+                <p>Sorry, we were unable to create this alert. Please <a href="mailto:<?= str_replace('@', '&#64;', CONTACTEMAIL) ?>">let us know</a>. Thanks.</p>
+            </div>
+
+          <?php else: ?>
+
+            <p class="lead">Enter your postcode, and we&rsquo;ll email you every time your MP speaks or receives a written answer.</p>
+
+          <?php if (isset($data['invalid-postcode-or-email'])): ?>
+            <div class="alerts-message alerts-message--error">
+                <h2>Oops!</h2>
+                <p>Please supply a valid UK postcode and email address.</p>
```

Comment:
Could this wording be changed just so it puts the onus more on us for e.g. missing new build postcodes and doesn't annoy the user (e.g. "not a known postcode", "we didn't recognise the postcode" or similar, as it may be our fault).